src/EnergyPlus/HVACHXAssistedCoolingCoil.cc

```diff
@@ -872,86 +724,75 @@ namespace HVACHXAssistedCoolingCoil {
                 }
 
             } else {
-                ShowWarningError(
-                    state, format("{}{}=\"{}\"", RoutineName, CurrentModuleObject, state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).Name));
-                ShowContinueError(
-                    state, format("Invalid {}=\"{}\"", cAlphaFields(4), state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).CoolingCoilType));
+                ShowWarningError(state, format("{}{}=\"{}\"", RoutineName, CurrentModuleObject, thisHXCoil.Name));
+                ShowContinueError(state, format("Invalid {}=\"{}\"", cAlphaFields(4), thisHXCoil.CoolingCoilType));
                 ErrorsFound = true;
             }
 
-            TestCompSet(state,
-                        state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).HXAssistedCoilType,
-                        state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).Name,
-                        state.dataLoopNodes->NodeID(SupplyAirInletNode),
-                        state.dataLoopNodes->NodeID(SecondaryAirOutletNode),
-                        "Air Nodes");
-
-            state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).HXAssistedCoilInletNodeNum =
-                GetOnlySingleNode(state,
-                                  state.dataLoopNodes->NodeID(SupplyAirInletNode),
-                                  ErrorsFound,
-                                  DataLoopNode::ConnectionObjectType::CoilSystemCoolingWaterHeatExchangerAssisted,
-                                  state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).Name,
-                                  DataLoopNode::NodeFluidType::Air,
-                                  DataLoopNode::ConnectionType::Inlet,
-                                  NodeInputManager::CompFluidStream::Primary,
-                                  ObjectIsParent);
-            CoolingCoilInletNodeNum = GetOnlySingleNode(state,
-                                                        state.dataLoopNodes->NodeID(SupplyAirOutletNode),
-                                                        ErrorsFound,
-                                                        DataLoopNode::ConnectionObjectType::CoilSystemCoolingWaterHeatExchangerAssisted,
-                                                        state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).Name,
-                                                        DataLoopNode::NodeFluidType::Air,
-                                                        DataLoopNode::ConnectionType::Internal,
-                                                        NodeInputManager::CompFluidStream::Primary,
-                                                        ObjectIsParent);
```

Comment:
[src/EnergyPlus/HVACHXAssistedCoolingCoil.cc:899]:(style),[unreadVariable],Variable 'CoolingCoilInletNodeNum' is assigned a value that is never used.

Reply:
@Myoldmopar I am suspecting these 2 GetOnlySingleNode calls that were removed. The nodes need to be at least registered. Unit tests probably wouldn't catch that.